pandas/core/internals.py

```diff
@@ -2467,6 +2440,10 @@ class DatetimeBlock(DatetimeLikeBlockMixin, Block):
     is_datetime = True
     _can_hold_na = True
 
+    # We add a dummy `tz` attribute here to make these block subclasses
```

Comment:
let's remove this too (and the tz propert and changes to box_func).

Reply:
OK.  Not a fan of the idea of de-duplicating or just don't want to it in this PR?